javaee/tests/src/test/java/org/jboss/forge/addon/javaee/cdi/CDIOperationsTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2013 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.jboss.forge.addon.javaee.cdi;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+
+import javax.inject.Inject;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.forge.addon.javaee.ProjectHelper;
+import org.jboss.forge.addon.javaee.cdi.ui.CDINewBeanCommand;
+import org.jboss.forge.addon.javaee.cdi.ui.CDINewQualifierCommand;
+import org.jboss.forge.addon.javaee.jpa.ui.JPANewEntityCommand;
+import org.jboss.forge.addon.projects.Project;
+import org.jboss.forge.addon.ui.controller.CommandController;
+import org.jboss.forge.addon.ui.result.Failed;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.test.UITestHarness;
+import org.jboss.forge.arquillian.AddonDependencies;
+import org.jboss.forge.arquillian.AddonDependency;
+import org.jboss.forge.arquillian.archive.AddonArchive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Unit tests to verify the behavior of the {@link CDIOperations} implementation class.
+ */
+@RunWith(Arquillian.class)
+public class CDIOperationsTest
+{
+   @Deployment
+   @AddonDependencies({
+            @AddonDependency(name = "org.jboss.forge.addon:ui-test-harness"),
```

Comment:
You can use just @AddonDependencies , Furnace will detect the dependencies from the tests' pom.xml